apps/server/src/shared/infra/identity-management/keycloak/keycloak.module.ts

```diff
@@ -0,0 +1,40 @@
+import { Module } from '@nestjs/common';
+import { Configuration } from '@hpi-schul-cloud/commons/lib';
+import { ConsoleWriterModule } from '@shared/infra/console';
+import KeycloakAdminClient from '@keycloak/keycloak-admin-client';
+import { KeycloakConsole } from './console/keycloak-management.console';
+import { KeycloakAdministrationService } from './keycloak-administration.service';
+import { IKeycloakSettings, KeycloakSettings } from './interface/keycloak-settings.interface';
+import { KeycloakManagementUc } from './uc/Keycloak-management.uc';
+import { KeycloakManagementController } from './controller/keycloak-management.controller';
+
+const keycloakConfiguration = (Configuration.get('FEATURE_IDENTITY_MANAGEMENT_ENABLED') as boolean)
```

Comment:
I think putting this in an extra file would be nicer. But it's not a must.
And check if this is the best practice to access configuration, please.

Reply:
Not sure if I got your intention here. Probably hiding the direct access to Configuration and string literal? With your suggestion of making this a dynamic module, this is the only occurrence of the feature flag. BTW Yes, that directly accessing the toggles or flags is the preferred way.

Reply:
The module takes care of providing services, objects,..., but the implementation of the provided injectables is in an extra file. So the module is providing a `KeycloakAdminClient` but the service's implementation is in an extra file. 
The keycloakConfiguration an object instead of a service, but I'd still prefer the definition of this object to be separated from the module.
Just a suggestion. Not a must.